src/views/reset-password/request-password-reset.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Card, Help, InputField, LogoIcon } from '@dhis2/ui'
+import { useAlert } from '@dhis2/app-service-alerts';
+import { useCallback, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useAuthAxios } from '../../hooks'
+import styles from './reset-password.module.css'
+import { RequestPasswordResetRequest } from "../../types";
+
+export const RequestPasswordReset = () => {
+    const navigate = useNavigate()
+    const [email, setEmail] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
+    const { show: showAlert } = useAlert(
+        ({ message }) => message,
+        ({ isCritical }) => (isCritical ? { critical: true } : { success: true })
+    )
+    const onInputChange = useCallback(({value}) => {
+        // Always clear the error when user starts typing again
+        setErrorMessage('')
+        setEmail(value)
+    }, [])
+    const [{ loading }, postRequestReset] = useAuthAxios<RequestPasswordResetRequest>(
+        {
+            method: 'POST',
+            url: '/users/request-reset',
+        },
+        { manual: true }
+    )
+
+    const onSubmit = useCallback(
+        async (event) => {
+            event.preventDefault()
+
+            if (!email) {
+                setErrorMessage('Please provide a valid email address')
+            }
+
+            try {
+                await postRequestReset({ data: { email } })
```

Comment:
I'd like to see [this](https://github.com/dhis2-sre/im-manager/pull/730/files#diff-ca91d4575c0561bc512e314d455708400a1e715c5c98a109cd79960b29ee8a3aR140) type used to validate the request body here.

Wouldn't `postResetPasswordRequest` be a more suitable name?

Reply:
I thought this is what's being [done here](https://github.com/dhis2-sre/im-web-client/pull/468/files#diff-b5485d49f49ba66f8cc79ab26921269241425c24dfae8153add1a8bb44993803R22). Can you give me an example?

The name of the request variable doesn't contain `Request` pretty much anywhere, so I thought that's how we do it. Renamed it to `postRequestPasswordReset`.